src/ralph/lib/mixins/fields.py

```diff
@@ -5,13 +5,13 @@
 from django.conf import settings
 from django.contrib.admin.widgets import AdminTextInputWidget
 from django.contrib.contenttypes.models import ContentType
-from django.core.exceptions import ValidationError
 from django.db import models
 from django.db.models.loading import get_model
 from django.forms.utils import flatatt
 from django.utils import six
 from django.utils.html import format_html, smart_urlquote
 from django.utils.translation import ugettext_lazy as _
+from rest_framework.exceptions import ValidationError
```

Comment:
It works like this:

<img width="1392" alt="screen shot 2017-03-15 at 16 09 02" src="https://cloud.githubusercontent.com/assets/214072/23955307/d1e18702-0999-11e7-9279-43777ff45a86.png">